src/cake.coffee

```diff
@@ -9,7 +9,12 @@ module.exports = (task) ->
   
     
   task 'build', ->
-    run "rm -rf lib && ./node_modules/.bin/coffee -o lib -c src && cd src && find . -name '*.js' -exec rsync -R {} ../lib \\;"
+    extensions = [ 'js', 'html', 'png', 'jpg', 'gif', 'css', 'md' ]
+    names = extensions.map (ext) ->
+      "-iname \\*.#{ext}"
+    find = "find . \\( #{names.join(' -o ')} \\)"
+    console.log("rm -rf lib && ./node_modules/.bin/coffee -o lib -c src && cd src && #{find} -exec rsync -R {} ../lib \\;")
```

Comment:
Assume this is intentionally announcing what's happening (vs. a debugging leftover)? Could also or alternatively add `-v`s to the `rm` and `rsync` to make them verbose, possibly.